src/helper_startup.py

```diff
@@ -128,6 +128,8 @@ def loadConfig():
 def updateConfig():
     """Save the config"""
     config = BMConfigParser()
+    if not config.has_option('bitmessagesettings', 'settingsversion'):
+        config.set('bitmessagesettings', 'settingsversion', 1)
     settingsversion = config.getint('bitmessagesettings', 'settingsversion')
```

Comment:
Better to just add a default value into `src/bmconfigparser.py`

Reply:
Adding it like this at least doesn't help:
```
    "bitmessagesettings": {
        "settingsversion": 1,
        "maxaddrperstreamsend": 500,
```

Reply:
Try using `safeGetInt` instead of `getint` in addition to adding the default value into the dict.

Reply:
switching to safegetint makes no difference whether settingsversion is included in BMConfigDefaults or not. No idea what the underlying cause is but my proposed fix adds only two lines...

Reply:
The code you give is one of the changes I tried with stated effect. To me looks like only error from tests is in Ui_MainWindow which isn't related to this as far as I can see. If you want me to make additional changes you'll have to be more specific than "rest the tests".